From 7846700bb8407fcb2d1e6d720783cbef3ba49e97 Mon Sep 17 00:00:00 2001
From: "Cardona Ruiz, Erick" <erick.cardona.ruiz@intel.com>
Date: Wed, 23 Mar 2016 17:19:24 -0600
Subject: [PATCH] remove sphinx<1.2

Signed-off-by: Cardona Ruiz, Erick <erick.cardona.ruiz@intel.com>
---
 test-requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test-requirements.txt b/test-requirements.txt
index 0c6e9fe..2f08481 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -9,7 +9,7 @@ nose
 nosexcover
 nosehtmloutput
 oslosphinx
-sphinx>=1.1.2,<1.2
+sphinx>=1.1.2
 os-testr>=0.4.1
 mock>=1.0
 python-swiftclient
-- 
2.5.0

